region/lowernorfair/east/Fast Pillars Setup Room.json

```diff
@@ -341,6 +341,43 @@
         "This strat requires heatproof because it's pretty chaotic to execute."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "No Power Bombs, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_heatProof",
+        "h_RModeCanRefillReserves",
+        {"or": [
+          {"and": [
+            "canBeLucky",
+            {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"]], "excludedWeapons": ["PowerBomb"]}}
```

Comment:
Patience and combat logic for charge only and slow kills.

Reply:
Exclude Morph Bombs

Reply:
Not sure if `canDodgeWhileShooting` is enough for slowbeam